app/views/layouts/student_course.html.erb

```diff
@@ -1,5 +1,6 @@
 <% content_for(:head) do %>
   <%= vite_javascript_tag 'student', nonce: true, media: 'all' %>
+  <title><%= @page_title  %></title>
```

Comment:
We shouldn't be adding it to the layout level, as this would mean every view should have this instance variable, which is against the current pattern we are following. Similar to the other students view, you can have the title attribute set in the respective view file.

I see that the view doesn't have a presenter, so it can be declared inplace in `app/views/courses/cohorts.html.erb`